integrations/octopus/.port/resources/blueprints.json

```diff
@@ -0,0 +1,292 @@
+[
+  {
+    "identifier": "octopusDeployProject",
+    "title": "Octopus Deploy Project",
+    "icon": "Octopus",
+    "description": "An Octopus Deploy project",
+    "schema": {
+      "properties": {
+        "slug": {
+          "type": "string",
+          "title": "Slug",
+          "description": "The slug identifier of the project"
+        },
+        "id": {
+          "type": "string",
+          "title": "Project ID",
+          "description": "The unique identifier of the project"
+        },
+        "url": {
+          "type": "string",
+          "title": "Project URL",
+          "format": "url",
+          "description": "The URL to access the project in Octopus Deploy"
+        },
+        "description": {
+          "type": "string",
+          "title": "Description",
+          "description": "The project description"
+        },
+        "spaceId": {
+          "type": "string",
+          "title": "Space ID",
+          "description": "The ID of the space in which the project exists"
+        },

```

Comment:
Shouldn't this be a relation to a space blueprint?

Reply:
It has been updated